client/settings/connect-stripe-account/index.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+import { __ } from '@wordpress/i18n';
+import { Button, Card } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import CardBody from '../card-body';
+import StripeBanner from '../../payment-method-icons/stripe-banner';
+
+const CardWrapper = styled( Card )`
+	max-width: 560px;
+
+	h2 {
+		font-size: 16px;
+	}
+
+	img {
+		width: 100%;
+	}
+`;
+
+const InformationText = styled.p`
+	color: #1e1e1e;
+`;
+
+const TermOfServicesText = styled.p`
+	color: #757575;
+	font-size: 12px;
+	font-weight: 400;
+	margin: 22px 0px 16px;
+`;
+
+const ButtonWrapper = styled.div`
+	align-items: center;
+	display: flex;
+	flex-wrap: wrap;
+
+	> :last-child {
+		box-shadow: none;
+
+		&:active,
+		&:focus,
+		&:hover {
+			box-shadow: none !important;
+			background: none !important;
```

Comment:
Is it possible to avoid using `!important` here and use selectors with a higher specificity instead?

Reply:
Your solution to this is good and no change needed here but I am wondering if we should maybe go with something like this instead (not sure if it's possible with `@emotion/styled` but it is with SCSS):

```
.button-wrapper {
  &#{&} {
    &#{&} {
      /* css here */
    }
  }
}
```

Essentially this results in this:

```
.button-wrapper.button-wrapper.button-wrapper { }
```